wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights Comments card

Fixes #10687 

---
Notes: 
- `View more` shows a temporary alert.
- Disclosure rows (Post and Pages) show a temporary alert when tapped.

---
To test:

- On a site with comments, go to Stats > Insights.
  - Tip: Hogwarts and en.blog are good examples.
- Scroll to `Comments`.
- Verify:
  - Pages and Posts show a disclosure, and Authors does not.
  - `No data yet` is displayed on tabs with no data.
  - `View more` is shown if there are more than 6 rows.
  - If neither tab has data, the `FilterTabBar` is hidden.

----
<img width="350" alt="authors" src="https://user-images.githubusercontent.com/1816888/50251838-5b213900-03a2-11e9-8566-473b1db19fb8.png">

---
<img width="350" alt="p p" src="https://user-images.githubusercontent.com/1816888/50251850-607e8380-03a2-11e9-9071-41aa6608fd41.png">

---
<img width="350" alt="no_comments" src="https://user-images.githubusercontent.com/1816888/50251952-ad625a00-03a2-11e9-9f15-ecd9337d4499.png">
